README.md

```diff
@@ -96,6 +96,9 @@ cd amplication
 2. Run the [setup script](https://github.com/amplication/amplication/blob/master/script/setup.ts), which takes care of installing dependencies, building packages and ensuring your workspace is dev-ready.
 
 ```bash
+# Due to ts-node dependency
+npm install -g ts-node # or just npm install
```

Comment:
I think this better be added to the prerequisite above (line 81). WDYT?

Reply:
Great point. I'll do asap.

Reply:
@overbit I don't see the change

Reply:
@yuval-hazaz forgot to push from my old laptop :-S